news/2017-12-21-owc-2017-concludes.md

```diff
@@ -0,0 +1,126 @@
+
+Here's what went down throughout the final battles:
+
+## Finals Weekend
+
+### South Korea vs Taiwan
+
+**South Korea: 7 - Taiwan: 6**
+
+Beginning with South Korea vs Taiwan, the endless rivalry between the two teams grew to a fever pitch, culminating in a colossal tiebreaker after some herculean effort from players on all sides. Pring and Pray from South Korea took turns carrying the team on their shoulders, while Flask and Rucker responded from Taiwan's side in kind.
+
+Poised to lose the tiebreaker-Tengaku in the wake of South Korea's earlier win against Brazil, Taiwan pulled out all the stops, sending Rucker in with a Hidden mod pick on the ultimate match of the set - scraping a hugely respectable 300k score, but still sadly losing the match overall to South Korea 7-6.
+
+Perhaps one of the most notable moments of the finals weekend, South Korea almost managed to scrape a win on AMAZING BREAK! with only ~28,000 score difference after a member of their team disconnected mid match. Outstanding effort.
+
+### United Kingdom vs Germany
+
+**United Kingdom: 7 - Germany: 1**
+
+Over the years the German roster has seen some monumental improvements and the birth of backbone carries like Dustice. The United Kingdom, fully determined in going for the very top, was not in the slightest interest of being stopped here. 
+
+Despite having been sent down to the Loser's Bracket by the Polish warriors in the weekend prior, the English roster recovered - knocking one decisive blow after another into the German defense line. Despite holding on to their best attempt, Dustice, imagaK, Neliel, and the rest could not muster up enough against the combined fury of Spare and Bubbleman – two of the United Kingdom's most prized circleclickers, ending the battle dominantly in favour of the United Kingdom 7-1.
+
+### United States vs Poland
+
+**Poland: 7 - United States: 6**
+
+Both strong contenders for the Champion title, the two teams felt tremendous pressure from each other. Team United States started off looking incredibly strong, taking 5 points in a row and producing FCs as if they were a cakewalk. 
+
+Just as everyone thought Team United States would finish the game in a flash and then walk on to claim their title 2 days later, idke's tablet pen inexplicably broke, sending Team United States' plan into disarray. Coupled with the wounded hand of their HardRock support player HappyStick, the US was suddenly painfully exposed to an explosive comeback from Team Poland. 
+
+And come back they did. Over the course of Poland's subsequent performance, there were at least 2 instances where Team United States could have packed the match in but failed to due to either players failing or just being slightly under in score. Everyone held their breaths as Team Poland, upon the 12th map of the match – Kokou no Sousei, clenched the absolute tiniest margin of victory by a mere 182 points.
```

Comment:
"upon the 12th map of the match – Kokou no Sousei,"
either use comma both ends, or dashes - you can't mix them in